index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+<body>
+    <header class="header">
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+                <i class="material-symbols-outlined menu_icon">
+                    menu
+                </i>
+            </nav>
+    </header>
+    <div class="container">
+        <h2>I’m Abdurahim Glad to see you!</h2>
```

Comment:
- [ ] I would suggest you please use this main title with `h1` tag because If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag